functional_tests/README.md

```diff
@@ -0,0 +1,35 @@
+# Functional Tests for Elliott
+
+This directory contains functional test suite for Elliott.
+The test suite works by invoking `elliott` as a subprocess then checking for
+the exit code and output.
+
+## Run Tests
+
+Currently the test suite doesn't mock any services, such as Errata Tool, Bugzilla, Brew, etc, but interacts with real production services. You have to obtain all required credentials before running the test suite:
+
+``` sh
+kinit $username
+bugzilla login
+# OCP pullspec in your $HOME/.docker/config.json
```

Comment:
Any possible for automating this part so we can put this functional test into CI?

Reply:
Yes we can. But considering it currently operates with real production services, it might have security concerns with real credentials configured. Better to have an integration test env I think.